client/ionic/ios/App/App/Info.plist

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleDisplayName</key>
+        <string>whoapp</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLName</key>
+			<string>com.getcapacitor.capacitor</string>
```

Comment:
TODO: Need our ID here.

Reply:
I will update this, but just leaving the note... this is registering our app to respond to URLs with the `capacitor://` scheme, and I don't think the URL identifier actually matters... I'm not even entirely sure why Ionic/Capacitor registers the scheme. but I suppose that it's possible that this would collide with other apps developed using Ionic, so I will change to be safe 👍

Reply:
I *think* we do need it to be unique for potential deeplinking but not sure.  https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app sg

Reply:
I think we would register a different URL scheme unique to us if we want to deep link, this is one just for Capacitor which I'm not sure exactly what it's doing... but regardless, it's updated as of ea9ea80 just in case 😄